cluster-autoscaler/cloudprovider/clusterapi/deployment.yaml

```diff
@@ -0,0 +1,158 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  namespace: kube-system
+  labels:
+    app: cluster-autoscaler
+spec:
+  selector:
+    matchLabels:
+      app: cluster-autoscaler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cluster-autoscaler
+    spec:
+      containers:
+      - image: us.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.18.1
+        name: cluster-autoscaler
+        command:
+        - /cluster-autoscaler
+        args:
+        - --cloud-provider=clusterapi
+        - --namespace=default
+        - --scale-down-enabled=true
+        - --scale-down-delay-after-add=10s
+        - --scale-down-delay-after-delete=10s
+        - --scale-down-delay-after-failure=10s
+        - --scale-down-unneeded-time=23s
```

Comment:
we might want to ease these scale down times back a little. i have used these aggressive timings for demonstration purposes but they are not suited for production. imo, we could drop these in favor of the default values (which are in the minutes), or perhaps change just put the defaults here so people can see them.